web/src/deb/resources/etc/georchestra/geonetwork/geonetwork.properties

```diff
@@ -1,3 +1,17 @@
+# geonetwork datadir subdirs
+geonetwork.dir=/path/to/your/geonetwork_data_dir
```

Comment:
/tmp/gn_data by default ?

(since this will end up as default configuration in the debian package, and logs are already redirected into /tmp)